### PR TITLE
Feature 77

### PR DESCRIPTION
**Pull request by w-liam-mcnair**
Added Herald for C

It is in working and stable condition.

### CODE REVIEW

File: herald-for-c/ble/BleDevice.c [hunk 1]
```diff
@@ -0,0 +1,138 @@
+#include "ble/BleDevice.h"
```
Comment (adamfowleruk) [inline herald-for-c/ble/BleDevice.c:1]: Missing SPDX copyright header

File: herald-wearable-c/src/herald.c [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+
+#include <zephyr.h>
+#include "herald.h"
+#include "payload/FixedPayloadSupplier.h"
+#include "logger/herald_logger.h"
+#include "abtt_payload.h"
+#include <random/rand32.h>
+
+static FixedPayloadSupplier_data_t fixedPayloadSupplier =
+    FixedPayloadSupplier_INIT(
```
Comment (adamfowleruk) [inline herald-wearable-c/src/herald.c:15]: These values should never be fixed - always passed in.

**Comment by adamfowleruk** [comment]
Looks good. If you can please add the missing SPDX header from the one file I can merge it all in.

File: herald-for-c/ble/BleDevice.c (hunk 1 above)
Comment (w-liam-mcnair) [reply herald-for-c/ble/BleDevice.c:7]: Added

**Comment by w-liam-mcnair** [comment]
I added the missing SPDX header. It should be good to go now

**Pull request merged**